### PR TITLE
[WIP] [DNM] Dump disk info on kubelet start

### PR DESCRIPTION
**Pull request by ehashman**
#### What type of PR is this?

/hold

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
why kubelet restarts 50 times?

> Jul 26 18:30:54 n1-standard-2-cos-81-12871-1290-12-80df8ac6 kubelet[24524]: I0726 18:30:54.373206   24524 configsync.go:204] "Kubelet config controller event" message="Kubelet restarting to use /api/v1/namespaces/kube-system/configmaps/testcfg-l4lzf, UID: ffd806f1-edc1-4190-a2b6-7851bba0bf16, ResourceVersion: 1971, KubeletConfigKey: kubelet"

**Comment by ehashman** [comment]
> why kubelet restarts 50 times?
> 
> > Jul 26 18:30:54 n1-standard-2-cos-81-12871-1290-12-80df8ac6 kubelet[24524]: I0726 18:30:54.373206   24524 configsync.go:204] "Kubelet config controller event" message="Kubelet restarting to use /api/v1/namespaces/kube-system/configmaps/testcfg-l4lzf, UID: ffd806f1-edc1-4190-a2b6-7851bba0bf16, ResourceVersion: 1971, KubeletConfigKey: kubelet"

I believe it's normal for these tests, DynamicKubeletConfig is updating the kubelet's config iirc

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -438,6 +439,11 @@ func Run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	logOption.Apply()
 	// To help debugging, immediately log version
 	klog.InfoS("Kubelet version", "kubeletVersion", version.Get())
+	cmd1 := osexec.Command("df")
+	cmd2 := osexec.Command("df", "-i")
+	out1, _ := cmd1.CombinedOutput()
+	out2, _ := cmd2.CombinedOutput()
+	klog.InfoS("Disk info", "space", out1, "inodes", out2)
```
Comment (aojea) [inline cmd/kubelet/app/server.go:446]:
```suggestion
go func(){
   for {
	cmd1 := osexec.Command("df")
	cmd2 := osexec.Command("df", "-i")
	out1, _ := cmd1.CombinedOutput()
	out2, _ := cmd2.CombinedOutput()
	klog.InfoS("Disk info", "space", out1, "inodes", out2)
	time.Sleep(15*time.Second)
	}
	}()
```

Comment (ehashman) [reply cmd/kubelet/app/server.go:446]: Could do that but I think we got to the bottom of it: https://github.com/kubernetes/kubernetes/issues/103879#issuecomment-887048157

Comment (aojea) [reply cmd/kubelet/app/server.go:446]: awesome, great investigation

**Pull request closed without merging**